src/DIRAC/WorkloadManagementSystem/Service/BasicPilotLoggingPlugin.py

```diff
@@ -0,0 +1,51 @@
+"""
+Basic Pilot logging plugin. Just log messages.
+"""
+from DIRAC import S_OK, S_ERROR, gLogger
+
+sLog = gLogger.getSubLogger(__name__)
+
+
+class BasicPilotLoggingPlugin:
+    """
+    This is a no-op fallback solution class, to be used when no plugin is defined for remote logging.
+    Any pilot logger plugin could inherit from this class to receive a set of no-op methods required by
+    :class:`TornadoPilotLoggingHandler` and only overwrite needed methods.
+    """
+
+    def __init__(self):
+
+        sLog.warn("BasicPilotLoggingPlugin is being used. It only logs locally at a debug level.")
+
+    def sendMessage(self, message, UUID, vo):
+        """
+        Dummy sendMessage method.
+
+        :param message: text to log
+        :type message: str
```

Comment:
You have several cases like this, where I would save a line with
```suggestion
        :param str message: text to log
```
Or, better, add type hints since this is only py3